e2e/test/FaultInjectionPoolAmqpTests.MethodFaultInjectionPoolAmqpTests.cs

```diff
@@ -1,13 +1,14 @@
-﻿// Copyright(c) Microsoft.All rights reserved.
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+// Copyright(c) Microsoft.All rights reserved.
```

Comment:
duplicate header?

Reply:
Good catch

Reply:
So this got added because of spacing differences in the existing header vs what is defined in our codemaid settings! 
All the more reason to not add these manually.